projects/plugins/jetpack/extensions/blocks/send-a-message/view.scss

```diff
@@ -11,6 +11,8 @@ div.wp-block-jetpack-whatsapp-button {
 		text-decoration: none;
 		white-space: nowrap;
 		min-height: 50px;
```

Comment:
While testing this block in the FSE site editor, I noticed that the `min-height: 50px` set here expects that the box sizing will be using `border-box`. If not, in the tt1-blocks theme, the block will be rendered like this within the editor:

![image](https://user-images.githubusercontent.com/14988353/113810260-32105680-97ad-11eb-86fd-3c5e1c527cad.png)

However, if we set the box-sizing for this component, it displays at the correct height:

![image](https://user-images.githubusercontent.com/14988353/113810480-b236bc00-97ad-11eb-869f-c5c9236df147.png)

Rather than expecting that the editor or theme sets the correct default behaviour, how about we add the following here:

```suggestion
		min-height: 50px;
		box-sizing: border-box;
```

Reply:
Added this in a separate follow-up: https://github.com/Automattic/jetpack/pull/19531 🙂